plugin/trino-exasol/src/test/java/io/trino/plugin/exasol/TestingExasolServer.java

```diff
@@ -0,0 +1,144 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.trino.plugin.exasol;
+
+import com.exasol.containers.ExasolContainer;
+import com.exasol.containers.ExasolService;
+import com.google.common.collect.ImmutableMap;
+import io.airlift.log.Logger;
+import io.trino.testing.ResourcePresence;
+import io.trino.testing.sql.JdbcSqlExecutor;
+import org.intellij.lang.annotations.Language;
+
+import java.io.Closeable;
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Map;
+import java.util.Properties;
+
+import static io.trino.testing.containers.TestContainers.startOrReuse;
+import static java.lang.String.format;
+
+public class TestingExasolServer
+        implements Closeable
+{
+    private static final Logger log = Logger.get(TestingExasolServer.class);
+    private static final String TEST_USER = "trino_test";
+    /**
+     * Name of the test schema. Must not contain an underscore, required by {@link io.trino.plugin.exasol.TestExasolConnectorTest#testShowSchemasLikeWithEscape()}
+     */
+    public static final String TEST_SCHEMA = "tpch";
+    private static final String TEST_PASS = "trino_test_password";
+
+    private final ExasolContainer<?> container;
+
+    private final Closeable cleanup;
+
+    public TestingExasolServer()
+    {
+        container = new ExasolContainer<>("8.26.0").withRequiredServices(ExasolService.JDBC);
+        container.withJdbcConnectionTimeout(600);
```

Comment:
is this "required"? If not let's remove it since actual Exasol users may not have such generous timeouts.